test.js

```diff
@@ -43,6 +51,13 @@ test('return empty array if non-valid file is supplied', async t => {
 	t.is(files.length, 0);
 });
 
+test('return empty array if non-valid file stream is supplied', async t => {
+	const stream = fs.createReadStream(__filename);
+	const files = await m()(stream);
+
+	t.is(files.length, 0);
```

Comment:
Should probably check the readable state of the stream here, do you know what is appropriate? I am thinking `stream._readableState.flowing === false` and `stream._readableState.ended === false`.

Reply:
implemented as described.